deployment/hasura/metadata/actions.yaml

```diff
@@ -4,131 +4,183 @@ actions:
       kind: synchronous
       handler: "{{AERIE_MERLIN_URL}}/addExternalDataset"
       timeout: 300
+    permissions:
+      - role: user
```

Comment:
Just to wrap my head around this - does this mean that `admin` and `viewer` cannot run these actions?

Reply:
`admin` can run everything. This means that `viewer` cannot run these actions.

Reply:
Thanks, that makes sense - where do we specify that `admin` can run everything?

Reply:
Ah, it seems that `admin` is a special role https://hasura.io/docs/latest/auth/authentication/admin-secret-access/#admin-role